ORBIT/phases/design/SemiTaut_mooring_system_design.py

```diff
@@ -0,0 +1,167 @@
+"""`MooringSystemDesign` and related functionality."""
+
+__author__ = "Jake Nunemaker, modified by Becca F."
+__copyright__ = "Copyright 2020, National Renewable Energy Laboratory"
+__maintainer__ = "Jake Nunemaker"
+__email__ = "jake.nunemaker@nrel.gov & rebecca.fuchs@nrel.gov"
+
+from scipy.interpolate import interp1d
+import numpy as np
+
+from ORBIT.phases.design import DesignPhase
+
+
+class SemiTautMooringSystemDesign(DesignPhase):
```

Comment:
Correct. The features were merged, but I decided to leave the duplicated files for now. I can add a deprecation warning on these, just in case a user is still implementing it. The hope is that when dev-> main, these will be removed.